src/sclient/tests/StorageClientTest.cpp

```diff
@@ -0,0 +1,167 @@
+#include <nebula/sclient/ScanEdgeIter.h>
+#include <nebula/sclient/StorageClient.h>
+#include "./ClientTest.h"
+
+// Require a nebula server could access
+
+static constexpr char kServerHost[] = "127.0.0.1";
+
+class StorageClientTest : public ClientTest {
+protected:
+    static void runGetSpaceVidType(nebula::StorageClient &c) {
+        std::cout << "get space vid type" << std::endl;
+        auto vidType = c.getSpaceVidType("nba");
+        std::cout << vidType << std::endl;
+        std::cout << "-----------------------" << std::endl;
+    }
+
+    static void runGetSpaceVidLen(nebula::StorageClient &c) {
+        std::cout << "get space vid len" << std::endl;
+        auto vidLen = c.getSpaceVidLen("nba");
+        std::cout << vidLen << std::endl;
+        std::cout << "-----------------------" << std::endl;
+    }
+
+    static void runGetParts(nebula::StorageClient &c) {
+        std::cout << "get parts of space" << std::endl;
+        auto parts = c.getParts("nba");
+        for (auto p : parts) {
+            std::cout << p << " ";
+        }
+        std::cout << std::endl;
+        std::cout << "-----------------------" << std::endl;
+    }
+
+    static void runGetPartsAlloc(nebula::StorageClient &c) {
+        std::cout << "get parts alloc of space" << std::endl;
+        auto partsAlloc = c.getPartsAlloc("nba");
+        for (auto p : partsAlloc) {
+            std::cout << "partReplicaSize[" << p.first << "]=" << p.second.size() << std::endl;
+        }
+        std::cout << "-----------------------" << std::endl;
+    }
+
+    static void runGetPartsLeader(nebula::StorageClient &c) {
+        std::cout << "get parts leader of space" << std::endl;
+        auto partsLeader = c.getPartsLeader("nba");
+        for (auto p : partsLeader) {
+            std::cout << "partLeader[" << p.first << "]=" << p.second << std::endl;
+        }
+        std::cout << "-----------------------" << std::endl;
+    }
+
+    static void runScanEdge(nebula::StorageClient &c) {
+        auto scanResultIter = c.scanEdgeWithPart("nba",
+                                                 999,
+                                                 "like",
+                                                 std::vector<std::string>(),
+                                                 1000,
+                                                 0,
+                                                 std::numeric_limits<int64_t>::max(),
+                                                 "",
+                                                 true,
+                                                 true);
+        {
+            EXPECT_EQ(scanResultIter.hasNext(), true);
+            nebula::DataSet ds = scanResultIter.next();
+            EXPECT_EQ(ds, nebula::DataSet());
+            EXPECT_EQ(scanResultIter.hasNext_, false);
+            EXPECT_EQ(scanResultIter.nextCursor_, "");
+        }
+
+        scanResultIter = c.scanEdgeWithPart("nba",
```

Comment:
Your test data is not inserted, so the test here is no used. Please perfect the test